charybdis-migrate/src/lib.rs

```diff
@@ -40,6 +40,11 @@ impl MigrationBuilder {
         self
     }
 
+    pub fn project_root(mut self, project_root: String) -> Self {
+        self.args.keyspace = project_root;
```

Comment:
@Michal-python Ah, I merged this with my phone, but missed this . We need to use `self.args.project_root` , not keyspace.